api/prisma/schema.prisma

```diff
@@ -145,6 +145,145 @@ model user {
 
 // -----------------------------------
 
+/// An exam as designed by the examiners
+model NewExam {
+  /// Globally unique exam id
+  ///
+  /// NOTE: Field is not "default", because of when/where exams are created.
+  id                 String         @id @map("_id") @db.ObjectId
+  /// Human-readable exam name
+  name               String
+  /// Notes given about exam
+  accessibility_note String
+  /// All questions for a given exam
+  question_types     QuestionType[]
+  /// Configuration for exam metadata
+  config             NewConfig
+}
+
+type QuestionType {
+  /// Unique question type id
+  id        String          @db.ObjectId
+  type      NewQuestionType
+  text      String?
+  questions NewQuestion[]
+}
```

Comment:
Should this be `QuestionGroup` or maybe `QuestionCategory`? It doesn't seem right to have `type` be a property of type.

Reply:
Maybe `QuestionCategory`, but that does not _ring as well with me_ as `QuestionType`.

Reply:
The issue is that QuestionType makes it sound like it's a question, but it contains questions. It also sounds like a type, but contains a type.

Reply:
It is one of the things that goes in `question_types`. It is a type. But, without putting the type on the item, within the array, how do we know the type?